me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/helper/ObjectHelper.java

```diff
@@ -82,4 +86,31 @@ public static String toStringByteArray(Object object) {
   public static boolean equals(Object a, Object b) {
     return (a == b) || (a != null && a.equals(b));
   }
+
+  /**
+   * This converts the InputStream to a String
+   *
+   * @param inputStream from the Web Request
+   * @return the converted string
+   */
+  public static String readStream(InputStream inputStream) throws IOException {
+    BufferedReader reader = null;
+    StringBuilder response = new StringBuilder();
+    try {
+      reader = new BufferedReader(new InputStreamReader(inputStream));
+      String line;
+      while ((line = reader.readLine()) != null) {
+        response.append(line);
+      }
+    } finally {
+      if (reader != null) {
+        try {
+          reader.close();
+        } catch (IOException exception) {
+          Log.e("Read stream error", exception.getMessage());
```

Comment:
I think that this exception is not actually useful. I would log instead any exception while reading the lines from the input stream. What do you think?